sections/Chapter 6 Sprint 4/Review.tex

```diff
@@ -0,0 +1,32 @@
+\section{Review}
+The goal for this sprint was primarily to catch up on documenting progress and previous sprints. As a side-goal, we needed to gather information about our next sprints, now that the Knox goal of implementing the search engine has concluded.
+
+\subsubsection*{What has been accomplished?}
+We managed to catch up on writing the documentation for the previous sprints. During our internal review, we found many areas which we are looking to improve in future sprints. These areas were discovered by performing a Premortem review of the documentation, which was a great way to get a sense of what we were missing. A Premortem is a method in which you imagine failure before you actually deliver the product, and you try to learn from those failures.
```

Comment:
There is no source on what a premortem is, or why it is a great way to get a sense of what is missing.

Reply:
Have changed the statement to be based upon our feelings ;) Which was meant from the start. And added citation.